relay/kusama/src/lib.rs

```diff
@@ -1902,6 +1906,50 @@ parameter_types! {
 	pub const MaxPoolsToMigrate: u32 = 500;
 }
 
+const NEW_MAX_POV: u32 = 10 * 1024 * 1024;
+
+pub struct Activate10MbPovs<T>(PhantomData<T>);
+impl<T> OnRuntimeUpgrade for Activate10MbPovs<T>
+where
+	T: parachains_configuration::Config,
+{
+	#[cfg(feature = "try-runtime")]
+	fn pre_upgrade() -> Result<Vec<u8>, sp_runtime::TryRuntimeError> {
+		// The pre-upgrade state doesn't matter
+		Ok(vec![])
+	}
+
+	fn on_runtime_upgrade() -> Weight {
+		match parachains_configuration::Pallet::<T>::set_max_pov_size(
```

Comment:
```suggestion
		match parachains_configuration::Pallet::<Runtime>::set_max_pov_size(
```

Then the struct doesn't need to be generic.

Reply:
I thought you loved generic structs :sweat_smile: 
Fixed in https://github.com/polkadot-fellows/runtimes/pull/553/commits/6e3353a0a20be5b862a67f88833b27cabb071073